File: app/src/main/java/com/doctor/testdemo/MainActivity.java
package com.doctor.testdemo;

import android.os.Bundle;
import android.widget.TextView;
import androidx.appcompat.app.AppCompatActivity;

public class MainActivity extends AppCompatActivity {
    private final String testString = "东方红太阳升";
    private final String testString1 = "东方红太阳升小红小白东方红太阳升小红小白东方红太阳升小红小白东方红太阳升小红小白东方红太阳升小红小白东方红太阳升小红小白东方红太阳升小红小白东方红太阳升小红小白东方红太阳升小红小白东方红太阳升小红小白东方红太阳升小红小白东方红太阳升小红小白东方红太阳升小红小白东方红太阳升小红小白东方红太阳升小红小白东方红太阳升小红小白东方红太阳升小红小白东方红太阳升小红小白";


    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        TextView textView = new TextView(this);
        textView.setText(testString);
        textView.setText(testString1);
        setContentView(textView);
    }
}

File: app/src/main/java/com/doctor/aistring/MainActivity.kt
package com.doctor.aistring

import android.os.Bundle
import android.widget.TextView
import androidx.appcompat.app.AppCompatActivity

class MainActivity : AppCompatActivity() {
    private val testString = "这是一个测试字符串"
    private val anotherString = "这是另一个测试字符串"

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        val textView = TextView(this)
        textView.text = testString + "\n" + anotherString
        setContentView(textView)
    }
}

